packages/core/src/scd-plugin/stand-alone-mode/index.ts

```diff
@@ -0,0 +1,44 @@
+// EMULATIONS
+import {
+	handleOpenDoc,
+	handleSaveDoc,
+	handleSimpleAction
+} from './edition-actions.stand-alone-mode'
+// TYPES
+import type {
+	PluginConstructor,
+	EditorPluginInstance,
+	PluginType
+} from '../types.scd-plugin'
+
+export default function standAloneMode(
+	pluginInstance: PluginConstructor,
+	pluginType: PluginType
+) {
+	customElements.define('stand-alone-plugin', pluginInstance)
+
+	//====== INITIALIZATION ======//
+
+	const pluginElement = document.createElement('stand-alone-plugin')
+	document.body.appendChild(pluginElement)
+
+	if (pluginType === 'editor')
+		emulateOpenSCDInstanceEditionActions(
+			pluginElement as EditorPluginInstance
+		)
+}
+
+//====== EMULATION ======//
+
+function emulateOpenSCDInstanceEditionActions(
```

Comment:
Is there a possibility of us activating the standalone mode multiple times on purpose or just accidentally? If so, creating and not disposing of these event handlers exposes us to memory leaks. Also the custom element remains in DOM even if mode changes

Reply:
Stand alone mode certainly needs improvements, this point should be kept in mind.